backend/app/Http/Controllers/Timeline/ShowAllTimelineController.php

```diff
@@ -18,6 +19,13 @@ class ShowAllTimelineController extends Controller
      */
     public function __invoke(): View|Factory
     {
-        return view('timeline.index', []);
+        $timelines = UserTimeline::with('User:id,name')
+            ->orderBy('start_date', 'desc')
+            ->take(10)
+            ->get();
+        foreach ($timelines as $timeline) {
+            $timeline->genreName = $timeline->genre->label();
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Access to an undefined property App\Models\UserTimeline::$genreName.